src/content/ui-api/kubectl-gs/template-cluster.md

```diff
@@ -122,6 +155,29 @@ kubectl gs template cluster \
   --label team=upstate
 ```
 
+{{< /tab >}}
+{{< tab id="command-examples-openstack" title="OpenStack">}}
+
+```nohighlight
+kubectl gs template cluster \
+  --provider openstack \
+  --enable-oidc \
+  --organization multi-project \
+  --failure-domain us-east-1 \
+  --cloud openstack \
+  --cloud-config cloud-config-giantswarm-2 \
+  --bastion-image-uuid 4b4bbad1-0eca-4937-9fe5-f5e767b36c5f \
```

Comment:
I assume these are random UUIDs and nothing "real"?

Reply:
Yes.

Reply:
In this case it would actually be nice to make that obvious, e. g. by using something like `12345678-abcd-1234-abcd-1234567890ab`

Reply:
I've updated to your suggested example UUID.